adapters/handlers/rest/configure_api.go

```diff
@@ -750,12 +750,17 @@ func configureAPI(api *operations.WeaviateAPI) http.Handler {
 			}
 		}, appState.Logger)
 	}
-	enterrors.GoWrapper(
-		func() {
-			ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
-			defer cancel()
-			backupScheduler.CleanupUnfinishedBackups(ctx)
-		}, appState.Logger)
+	suppressCleanup := os.Getenv("WEAVIATE_SUPPRESS_CLEANUP") == "true"
```

Comment:
can we use `entcfg.Enabled(...)` helper method to check if `WEAVIATE_SUPPRESS_CLEANUP` is set? and can we maybe rename it to `DISABLE_CLEANUP_UNFINISHED_BACKUPS`